docker/Dockerfile.windows

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.17
```

Comment:
you can use `plugins/base:windows-ltsc2022-amd64` for windows variant

Reply:
Done